neo/Ledger/SendersFeeMonitor.cs

```diff
@@ -0,0 +1,96 @@
+using Neo.Network.P2P.Payloads;
+using System.Collections.Generic;
+using System.Numerics;
+using System.Threading;
+
+namespace Neo.Ledger
+{
+    public class SendersFeeMonitor
```

Comment:
Nice job, @Qiao-Jin!

How about instanciate the `SendersFeeMonitor` both in the `Mempool` and in the `ConsensusContext`, then, we would not need two `ReaderWriterLockSlim` and `Dictionary<UInt160, BigInteger>`.
A generic class that we just instantiate the object.